server/database/get-user-data.js

```diff
@@ -0,0 +1,33 @@
+const dbClient = require('./db_connection.js');
+const formatUserData = require('./format-user-data');
+
+const getGoals = `
+  SELECT
+    goals.user_id,
+    goals.goal_id,
+    goals.title,
+    goals.icon,
+    ratings.rating_id,
+    ratings.rating,
+    ratings.comment,
+    ratings.date_created
+  FROM (goals LEFT OUTER JOIN ratings ON goals.goal_id = ratings.goal_id)
+  WHERE
+    goals.user_id=$1 
+  AND
+    goals.deleted=false
+  GROUP BY goals.goal_id, ratings.rating_id
+  ORDER BY goals.goal_id, ratings.rating ASC
+`;
```

Comment:
Just realised that we're not getting the created date for the goals! This is then missing from the frontend. As far as I can see, the goals are being ordered by their ID, which is not incremental. Therefore, goals could end up getting listed in a seemingly random order. I think we need to include the created date here, and order by date created.